name,value,relevant log,explanation
io.bytes.per.checksum,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.namenode.fs-limits.max-xattrs-per-inode,32,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.client.read.short.circuit.replica.stale.threshold.ms,1800000,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",property-item-hit	read
hadoop.security.key.default.bitlength,128,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",property-item-hit	default
mapreduce.jobhistory.loadedjobs.cache.size,5,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
io.bytes.per.checksum,0,"ERROR	Task: attempt_1701312551309_0713_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.namenode.fs-limits.max-xattrs-per-inode,32,"ERROR	Task: attempt_1701312551309_0713_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
hadoop.security.key.default.bitlength,128,"ERROR	Task: attempt_1701312551309_0713_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	default
mapreduce.jobhistory.loadedjobs.cache.size,5,"ERROR	Task: attempt_1701312551309_0713_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
io.bytes.per.checksum,0,"INFO	Diagnostics report from attempt_1701312551309_0713_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.namenode.fs-limits.max-xattrs-per-inode,32,"INFO	Diagnostics report from attempt_1701312551309_0713_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
hadoop.security.key.default.bitlength,128,"INFO	Diagnostics report from attempt_1701312551309_0713_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	default
mapreduce.jobhistory.loadedjobs.cache.size,5,"INFO	Diagnostics report from attempt_1701312551309_0713_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.jobhistory.loadedjobs.cache.size,5,INFO	1 failures on node cf6e5ca2712c,value-hit
io.bytes.per.checksum,0,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
mapreduce.jobhistory.loadedjobs.cache.size,5,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	jobhistory
io.bytes.per.checksum,0,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
mapreduce.jobhistory.loadedjobs.cache.size,5,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	jobhistory
io.bytes.per.checksum,0,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
